dpti/ti.py

```diff
@@ -919,6 +920,36 @@ def compute_task(job, inte_method, Eo, Eo_err, To, scheme="simpson"):
     return info
 
 
+def run_task(task_name, machine_file):
+    task_dir_list = glob.glob(os.path.join(task_name, "task.*"))
+    task_dir_list = sorted(task_dir_list)
+    work_base_dir = os.path.join(os.getcwd())
```

Comment:
**Suggestion:** Consider using `os.path.abspath()` instead of `os.getcwd()` to get the absolute path of the current directory. This will ensure that the path is always correct, regardless of where the script is run from. [enhancement]
```suggestion
    work_base_dir = os.path.abspath('.')
```

Reply:
This seems to be wrong.

Reply:
> This seems to be wrong.

I think this looks correct?

Reply:
I mean, the AI's suggestion looks wrong. `os.getcwd()` should give the same result as `os.path.abspath('.')`. However, I don't understand why you call `os.path.join`